projects/skia/Dockerfile

```diff
@@ -56,6 +56,8 @@ RUN wget -O $SRC/skia/encoder_seed_corpus.zip https://storage.googleapis.com/ski
 
 RUN wget -O $SRC/skia/skottie_json_seed_corpus.zip https://storage.googleapis.com/skia-fuzzer/oss-fuzz/skottie_json_seed_corpus.zip
 
+RUN wget -O $SRC/skia/skjson_seed_corpus.zip https://storage.googleapis.com/skia-fuzzer/oss-fuzz/skjson_seed_corpus.zip
```

Comment:
@kjlubick I'm sorry to bother you, but I was looking for a corpus for fuzzing a json parser being introduced in https://github.com/systemd/systemd/pull/9762. Would it be OK if I started to use this one?

Reply:
Sure.  Good luck!